lib/testing/src/lib/protractor/core/utils/date-util.ts

```diff
@@ -15,15 +15,16 @@
  * limitations under the License.
  */
 
-import * as moment from 'moment';
+import { addDays } from 'date-fns';
+import { DateFnsUtils } from '../../../../../../core/src/lib/common/utils/date-fns-utils';
 
 export class DateUtil {
 
-    static formatDate(dateFormat: string, date: Date = new Date(), days: number | string = 0): string {
-        return moment(date).add(days, 'days').format(dateFormat);
+    static formatDate(dateFormat: string, date: Date = new Date(), days: number = 0): string {
```

Comment:
Don't remove the string from the signature. convert it into a number if the string is coming.
if(typeOf days === 'string') {
 days = parseInt(days);
}

Reply:
Done